src/test/java/teammates/test/cases/webapi/SearchStudentsActionTest.java

```diff
@@ -0,0 +1,145 @@
+package teammates.test.cases.webapi;
+
+import org.junit.Test;
+
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
+import teammates.ui.webapi.action.JsonResult;
+import teammates.ui.webapi.action.SearchStudentsAction;
+import teammates.ui.webapi.output.SearchStudentsData;
+
+/**
+ * SUT:{@link SearchStudentsAction}.
+ */
+public class SearchStudentsActionTest extends BaseActionTest<SearchStudentsAction> {
+
+    @Override
+    protected void prepareTestData() {
+        DataBundle dataBundle = getTypicalDataBundle();
+        removeAndRestoreDataBundle(dataBundle);
+        putDocuments(dataBundle);
+    }
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.STUDENTS_SEARCH;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @Override
+    protected void testExecute() {
+        // TODO: Remove
```

Comment:
testExecute as an abstract method on the parent class and is present in quite a bit of classes. Will refactor in a separate PR as its not critical now